notebooks/softvc-voice-conversion/softvc-voice-conversion.ipynb

```diff
@@ -40,7 +40,26 @@
    "metadata": {},
    "source": [
     "## Prerequisites\n",
-    "[back to top ⬆️](#Table-of-contents:)\n"
+    "[back to top ⬆️](#Table-of-contents:)\n",
+    "This model has some problems with installation. So, we have to downgrade `pip` to version below `24.1` to avoid problems with `fairseq`. Also, `fairseq` has some problems on `Windows` on `python>=3.11` and we use a custom version of this library to resolve them.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2d4dacb3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import platform\n",
+    "import sys\n",
+    "\n",
+    "\n",
+    "%pip install -qU \"pip<24.1\"\n",
+    "if platform.system() == \"Windows\" and sys.version_info >= (3, 11, 0):\n",
+    "    %pip install git+https://github.com/aleksandr-mokrov/fairseq.git\n",
```

Comment:
can you please also add warning here that it may be time consuming and if you would like avoid that please downgrade python?